packages/i18n/data/core.json

```diff
@@ -105,8 +105,8 @@
   "QuickAccess.openOrdersList": "Open Orders List",
   "QuickAccess.openPimSearch": "Open PIM Search",
   "QuickAccess.openProductDiscountsList": "Open Product Discounts List",
-  "QuickAccess.openProductList": "Open Product List",
   "QuickAccess.openProducts": "Open Products",
+  "QuickAccess.openProductsList": "Open Products List",
```

Comment:
Is someone taking care of updating this in TX as well?

Reply:
These keys will be passed on to UX for updating in transifex after this ticket is merged

Reply:
Wouldn't it be better to edit the existing translation instead of creating a new one?

Reply:
usually would edit the existing translation, but I think a new key would be more appropriate in this case because the key is the same verbiage as the text